lib/hydra/works/services/virus_checker_service.rb

```diff
@@ -27,30 +27,10 @@ def file_has_virus?
 
     private
 
-      # Stubbing out the behavior of "The Clam" was growing into a rather nasty
-      # challenge. So instead I'm injecting a system scanner. This allows me to
-      # now test the default system scanner in isolation from the general response
-      # to a system scan.
-      def default_system_virus_scanner
-        if defined?(ClamAV)
-          ClamAV.instance.method(:scanfile)
-        else
-          lambda do |_path|
-            :no_anti_virus_was_run
-          end
-        end
-      end
-
       def handle_virus_scan_results(path, scan_result)
-        case scan_result
-        when 0 then return false
-        when 1
-          warning("A virus was found in #{path}: #{scan_result}")
-          true
-        else
-          warning "Virus checking disabled, #{path} not checked"
-          true
-        end
+        return false if scan_result == 0
```

Comment:
The `0` here is a ClamAV thing. Should we instead expect the new `VirusScanner` class's `#call` method to return a boolean so that we don't need to bake scanner-specific logic into other code?